cmd/oras/internal/option/packer.go

```diff
@@ -99,8 +99,10 @@ func (opts *Packer) LoadManifestAnnotations() (annotations map[string]map[string
 	}
 	if opts.AnnotationFilePath != "" {
 		if err = decodeJSON(opts.AnnotationFilePath, &annotations); err != nil {
-			return nil, err
+			return nil, fmt.Errorf("error loading manifest annotations from file %s: %w " + 
+				"refer to the document at https://oras.land/docs/how_to_guides/manifest_annotations", opts.AnnotationFilePath, err)
```

Comment:
Should remove the `manifest` since maybe the error comes from the layer annotations.
```suggestion
			return nil, fmt.Errorf("failed to load annotations from %s: %w " + 
				"Please refer to the document at https://oras.land/docs/how_to_guides/manifest_annotations", opts.AnnotationFilePath, err)
```